live-examples/js-examples/statement/statement-async-function-asterisk.html

```diff
@@ -0,0 +1,22 @@
+<pre>
+<code id="static-js" data-height="taller">async function* randomNumbers() {
+  const url = 'https://www.random.org/decimal-fractions/?num=1&dec=10&col=1&format=plain&rnd=new';
+
+  console.log("Print randomly generated numbers until reset is clicked the random number exceeds 0.95.");
```

Comment:
Looks to be a small typo here:

`"Print randomly generated numbers until reset is clicked the random number exceeds 0.95."`

to

`"Print randomly generated numbers until reset is clicked, (or) the random number exceeds 0.95."`

Reply:
@jpmedley No worries, let me fix that one, and merge. Thanks again for your contribution.